libavcodec/x86/vvc/vvcdsp_init.c

```diff
@@ -217,6 +229,7 @@ void ff_vvc_dsp_init_x86(VVCDSPContext *const c, const int bd)
         }
         if (EXTERNAL_AVX2_FAST(cpu_flags)) {
             MC_LINKS_AVX2(8);
+            SAD_INIT();
```

Comment:
I think the asm can only work for 16bpc? not 8bpc, right?

Reply:
Example

```
conformance/passed/v1/8b420_A_Bytedance_2.bit

Samples: 14K of event 'cycles:Pu', Event count (approx.): 4030503793
  Children      Self  Command       Shared O  Symbol
+    0.80%     0.80%  vf#0:0        ffmpeg_g  [.] ff_sad16_sse2
     0.62%     0.62%  ffmpeg_g      ffmpeg_g  [.] ff_vvc_sad_16_16bpc_avx2
     0.22%     0.22%  vf#0:0        ffmpeg_g  [.] ff_sad8_mmxext
     0.21%     0.21%  vf#0:0        ffmpeg_g  [.] sad_hpel_motion_search
```